CHANGELOG.md

```diff
@@ -7,11 +7,19 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Enhancements
 
-* Swift: A `write` block may now `throw`, reverting any changes already made in the transaction.
+* Swift: A `write` block may now `throw`, reverting any changes already made in
+  the transaction.
+* Reduce address space used when committing write transactions.
+* Significantly reduce the download size of prebuilt binaries and slightly
+  reduce the final size contribution of Realm to applications.
 
 ### Bugfixes
 
 * Fix a crash when reading the shared schema from an observed Swift object.
+* Fix a crash when deleting all objects in a Realm while fast-enumerating query
+  results from that Realm.
+* Handle EINTR from flock() rather than crashing.
+* Fix incorrect behavior following a call to `[RLMRealm compact]`.
```

Comment:
It's not actually public (although a few users have used it anyway) and we didn't get any reports of problems with it, so a wall of text about a problem that probably never effected any of the people reading this doesn't seem useful.